public/static/docs/command-reference/metrics/modify.md

```diff
@@ -1,7 +1,7 @@
 # metrics modify
 
-Modify [project metric](/doc/command-reference/metrics) values such as type,
-path expression that is used to parse it, etc. (See full options below.)
+Modify [project metric](/doc/command-reference/metrics) default formatting with
```

Comment:
hmm ... I kinda like it, but we might need to change it later ... it's not about formatting only eventually

Reply:
I understand. I see that the original intention was probably to add more options than type and xpath but for now its only about formatting so it makes sense I think, and yes of course we'll have to update later if more options are added that change other kind of definitions.

Reply:
But if you prefer I can try to use a different term like "definition" or "specification".

Reply:
not a strong opinion, let's see what other guys on the DVC core say about this ... I like actually how does it sound now - it's very user friendly